components/sidebar/mobile_sidebar_header/contents/contents.tsx

```diff
@@ -33,31 +30,30 @@ const VerticalStack = styled.div`
 `;
 
 type Props = {
-    showTutorialTip: boolean;
     teamDescription: string;
     teamId: string;
     currentUser: UserProfile;
     teamDisplayName: string;
+    actions: Actions;
+};
+
+type Actions = {
     openModal: <P>(modalData: ModalData<P>) => void;
 };
 
-export default class SidebarHeaderDropdownButton extends React.PureComponent<Props> {
+export default class Contents extends React.PureComponent<Props> {
```

Comment:
Not for this PR, but this one looks like a good candidate to migrate to functional component. Looks pretty simple and straight forward.